pyds8k/resources/ds8k/v1/cs/flashcopies.py

```diff
@@ -0,0 +1,68 @@
+##############################################################################
+# Copyright 2019 IBM Corp.
```

Comment:
shouldn't it be 2020?

Reply:
I'm not getting into legals.. if it need to be change I will change it, but I really dont have any clue..

Reply:
ok